### ISSUE TITLE
Migrate checklists to GitHub Trilinos Developer site

### ISSUE THREAD

**Issue by jwillenbring** [trilinos/Trilinos#473]
**Next Action Status:**

Updating the new developer checklist ...

**CC:** @trilinos/framework 

**Description:**

The process checklists need to be moved from the old developer website to the new Trilinos Developer GitHub wiki.

**Tasks:**
1. Update the new-developer checklist ... IN PROGRESS..
2. ???

**Comment by bartlettroscoe** [comment]
Jim, given the emails from last night, I think this is already in progress, right?  Any reason not to log our emails here and the different versions of the new-developers checklist?

**Comment by jwillenbring** [comment]
I have issue #372 in progress for that work. I viewed this ticket as being a wholesale move of existing checklists, and 372 as being a single task focused more on the update than the move (but why not move it too?). I don't intend to move all of the other checklists imminently, so I would prefer not to put this in progress.

**Comment by bartlettroscoe** [comment]
> I have issue #372 in progress for that work. I viewed this ticket as being a wholesale move of existing checklists, and 372 as being a single task focused more on the update than the move (but why not move it too?). I don't intend to move all of the other checklists imminently, so I would prefer not to put this in progress.

Okay, I see two different options then:
1. Change this issue "Migrate checklists to GitHub Trilinos Developer site" into a GitHub milestone/epic and then stories for each checklist like #372 be put in that epic.
2. Leave this issue as is and make it clear in the description field that this is being blocked by #372 and other issue tickets.

**Comment by bartlettroscoe** [comment]
Note that I was never CCed (by adding @trilinos/framework) on #372 so I did not even know that it existed.  I just happened to stumble on this current issue ticket by accident.

We need to use mentions as described [here](https://github.com/trilinos/Trilinos/wiki/New-Issue-Cheat-Sheet) (see "Mention Associated Teams and Developers") for GitHub Issues to be an effective an effective project management and communication tool.  That is the offical policy that we both worked on so let's do it.

We also need to standardize the fields "Blocked By", "Blocking" and "Relates To" (listed in the first comment).
